payment-cards/pci-compliance/front-end.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: PCI on the Front-End
```

Comment:
My only hesitation is that Client-Side sounds like it encompasses native mobile when this page is specifically talking about browsers.